docs/config-file/v2.rst

```diff
@@ -427,6 +453,60 @@ Do a recursive clone of the submodules.
 
    This is ignored if there aren't submodules to clone.
 
+search
+~~~~~~
+
+Settings for more control over :doc:`/server-side-search`.
+
+.. code-block:: yaml
+
+   version: 2
+
+   search:
+      ranking:
+        api/v1/*: -1
+        api/v2/*: 4
+
+search.ranking
+``````````````
```

Comment:
We can mark this option as experimental if we want to try it first, but we can make changes without re-indexing, so we are safe to do some tuning after if needed.